deploy/crds/kieapp.crd.yaml

```diff
@@ -97,45 +93,261 @@ spec:
                 server:
                   type: object
                   description: Configuration of the RHPAM KIE server
-#                  anyOf:
-#                  - required:
-#                    - env
-#                  - required:
-#                    - resources
                   properties:
-                    env:
-                      type: array
-                      items:
+                    deployments:
+                      type: integer
+                      format: int16
+                      description: Number of Server sets that will be deployed
+                    from:
+                      type: object
+                      description: Image definition to use for all the servers
+                      required:
+                        - kind
+                        - name
+                      properties:
+                        kind:
+                          type: string
+                          description: Object kind. e.g. ImageStreamTag
```

Comment:
Any other examples? Should this be an enum?

Reply:
Sure, I will limit it to just DockerImage and ImageStreamTag